src/main/java/com/yolo/customer/idea/IdeaController.java

```diff
@@ -17,15 +26,64 @@ public IdeaController(IdeaService ideaService) {
     }
 
     @PatchMapping("/{ideaId}")
```

Comment:
PreAuthorize annotation missing

Reply:
Ahmad's api. Review by Ahmad is required

Reply:
@Ahmad-981